src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT04-23-expected.json

```diff
@@ -3,10 +3,10 @@
   "type": "transaction",
   "entry": [
     {
-      "fullUrl": "urn:uuid:18548d54-14a8-815a-c1bb-396f35b37164",
+      "fullUrl": "urn:uuid:",
```

Comment:
Why do we remove id for this test case?

Reply:
Might because of the copying issue, I'll fix it, thx.